src/fourierdomain/nuft/nfft_alg.jl

```diff
@@ -107,6 +107,45 @@ end
     return nothing
 end
 
+function forward(
+    config::EnzymeRules.FwdConfig,
+    func::Const{typeof(_jlnuft!)}, 
+    RT, 
+    out::Duplicated, A::Const{<:NFFTPlan},
+    b::Duplicated{<:AbstractArray{<:Real}}
+) 
+    if EnzymeRules.needs_primal(config) && EnzymeRules.needs_shadow(config)
+        if EnzymeRules.width(config) == 1
+            func.val(out.dval, A, b.dval)
```

Comment:
```suggestion
        func.val(out.dval, A, b.dval)
        if EnzymeRules.width(config) == 1
```
let's move this out of the if statement just for clarity.